es5-shim.js

```diff
@@ -329,6 +360,97 @@ var strSplit = call.bind(StringPrototype.split);
 var strIndexOf = call.bind(StringPrototype.indexOf);
 var push = call.bind(array_push);
 
+// ES-5 15.3.4.4
+// http://es5.github.io/#x15.3.4.4
+var hasToStringTagBasicBug = !(toStr() === '[object Undefined]' && toStr(null) === '[object Null]');
+var hasToStringTagArgumentsBug = toStr(function () {
+  return arguments;
+}) !== '[object Arguments]';
```

Comment:
this isn't a bug so much as a legacy arguments object - which is already checked [here](https://github.com/es-shims/es5-shim/blob/master/es5-shim.js#L932-L945)